src/Shared/Process/ProcessEx.cs

```diff
@@ -180,27 +174,22 @@ public void WaitForExit(bool assertSuccess, TimeSpan? timeSpan = null)
         {
             if(!timeSpan.HasValue)
             {
-                timeSpan = TimeSpan.FromSeconds(600);
+                timeSpan = TimeSpan.FromSeconds(480);
             }
 
-            var exited = Exited.Wait(timeSpan.Value);
-            if (!exited)
-            {
-                _output.WriteLine($"The process didn't exit within the allotted time ({timeSpan.Value.TotalSeconds} seconds).");
-                _process.Dispose();
-            }
-            else if (assertSuccess && _process.ExitCode != 0)
+            Exited.Wait(timeSpan.Value);
+
+            if (assertSuccess && _process.ExitCode != 0)
             {
                 throw new Exception($"Process exited with code {_process.ExitCode}\nStdErr: {Error}\nStdOut: {Output}");
             }
         }
 
-        private static string GetNugetPackagesRestorePath() => (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("NUGET_RESTORE"))) 
-            ? typeof(ProcessEx).Assembly
+        private static string GetNugetPackagesRestorePath() =>
+            typeof(ProcessEx).Assembly
```

Comment:
merge issues in this file, does this break template tests in helix?

Reply:
Ah yes